lib/systems/inspect.nix

```diff
@@ -5,15 +5,18 @@ with lib.lists;
 
 rec {
   patterns = rec {
-    "32bit"      = { cpu = { bits = 32; }; };
-    "64bit"      = { cpu = { bits = 64; }; };
     i686         = { cpu = cpuTypes.i686; };
     x86_64       = { cpu = cpuTypes.x86_64; };
     PowerPC      = { cpu = cpuTypes.powerpc; };
     x86          = { cpu = { family = "x86"; }; };
     Arm          = { cpu = { family = "arm"; }; };
     Aarch64      = { cpu = { family = "aarch64"; }; };
     Mips         = { cpu = { family = "mips"; }; };
+    Riscv        = { cpu = { family = "riscv"; }; };
```

Comment:
Or do we do `RiscV`? :D

Reply:
The official style seems to be "RISC-V" -- if that doesn't fit the mold, I suppose "RiscV" at least captures the separation of the "RISC" and "V", so "RiscV" seems like a decent compromise and might be better than "Riscv".

With that said, I'm not adamant about it.

Reply:
Convinced!